documentation/business_care_intitule.md

```
@@ -0,0 +1,369 @@
+﻿*PROJET ANNUEL 2 ESGI i* 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)